eng/tox/run_sphinx_apidoc.py

```diff
@@ -31,33 +32,42 @@ def copy_existing_docs(source, target):
         logging.info("Copying {}".format(file))
         shutil.copy(os.path.join(source, file), target)
 
-def sphinx_apidoc(working_directory):
+def sphinx_apidoc(working_directory, namespace):
```

Comment:
Super nit and feel free to ignore. Any time I touch a function, if it's untyped, I `type` it.

```suggestion
def sphinx_apidoc(working_directory: str, namespace: str) -> None:
```